src/conf.c

```diff
@@ -57,6 +57,19 @@
 
 int parse_interfaces( conf_t *conf, char *s );
 
+#define axel_fscanf( _fp, ... )				\
+	{						\
+		int ret = fscanf( _fp, __VA_ARGS__ );	\
+		if( ret == EOF  && errno != 0 )		\
```

Comment:
`ferror` must be called instead of checking `errno`, there's no requirement for the function to set `errno` to zero.

Reply:
mh, right. I misinterpreted the manpage I think.